blog/zh/blog/2022/10/28/apisix-ingress-with-horizon-ai.md

```diff
@@ -0,0 +1,126 @@
+---
+title: "从 Traefik 到 APISIX，汽车智能计算平台公司「地平线」在 Ingress Controller 的探索和实践"
+authors:
+  - name: "张昕"
+    title: "Author"
+  - name: "韩飞"
+    title: "Technical Writer"
+    url: "https://github.com/hf400159"
+    image_url: "https://github.com/hf400159.png"
```

Comment:
Please add the cover for this user case, see https://github.com/apache/apisix-website/issues/1245#issuecomment-1294425812.

Reply:
done